### PR TITLE
Feat/documentation

### PR DESCRIPTION
**Pull request by simonovic86**
- add js-doc documentation
- generate and add API documentation
- update dependencies

**NOTE**: minor changes to `README.md`(s) documentation. Still WIP.

### CODE REVIEW

**Review by PaulLeCam — APPROVED** [review]

File: packages/ceramic-http-client/README.md [hunk 1]
```diff
@@ -13,10 +13,20 @@ $ npm install @ceramicnetwork/ceramic-http-client
 
 ```
 import Ceramic from '@ceramicnetwork/ceramic-http-client'
+import IdentityWallet from 'identity-wallet'
 
-// TODO: DEMONSTRATE API
+const API_URL = "http://localhost:7007"
+
+const client = new CeramicClient(API_URL)
```
[the remark this answers is not in the document]
Comment (simonovic86) [reply packages/ceramic-http-client/README.md:20]: yep, tnx! I need to update all the README.md(s) more

**Review by oed — APPROVED** [review]
Should be good. I'll turn on the github pages thing on develop!

**Comment by oed** [comment]
Should appear on https://ceramicnetwork.github.io/js-ceramic/

You might need to configure this as well: https://github.com/TypeStrong/typedoc/issues/620

**Comment by simonovic86** [comment]
> Should be good. I'll turn on the github pages thing on develop!

awesome! tnx 👍

**Comment by simonovic86** [comment]
> Should appear on https://ceramicnetwork.github.io/js-ceramic/
> 
> You might need to configure this as well: [TypeStrong/typedoc#620](https://github.com/TypeStrong/typedoc/issues/620)

👍

**Pull request merged**